Auth0/IDTokenValidatorContext.swift

```diff
@@ -45,7 +45,7 @@ struct IDTokenValidatorContext: IDTokenSignatureValidatorContext, IDTokenClaimsV
     }
 
     init(authentication: Authentication, leeway: Int, nonce: String?, maxAge: Int?) {
-        self.init(issuer: authentication.url.host!,
+        self.init(issuer: "\(authentication.url.absoluteString)/",
```

Comment:
**Question:** Can you tell me a bit about what `authentication.url.absoluteString` provides?

Reply:
It provides the absolute string for the `URL`. See https://developer.apple.com/documentation/foundation/url/1779984-absolutestring.